src/framework/shortcuts/internal/shortcutscontroller.h

```diff
@@ -24,13 +24,15 @@
 #include "ishortcutsregister.h"
 #include "actions/iactionsdispatcher.h"
 #include "actions/iactionsregister.h"
+#include "context/iglobalcontext.h"
 
 namespace mu::shortcuts {
 class ShortcutsController : public IShortcutsController
 {
     INJECT(shortcuts, IShortcutsRegister, shortcutsRegister)
     INJECT(shortcuts, actions::IActionsDispatcher, dispatcher)
     INJECT(shortcuts, actions::IActionsRegister, aregister)
+    INJECT(shortcuts, context::IGlobalContext, globalContext)
```

Comment:
The framework will depend on an external module after this change. Probably the issue should be fixed in the notation module

Reply:
Ok, will fix it.